.circleci/local_publish_helpers.sh

```diff
@@ -198,3 +198,25 @@ function emitCanaryFailureMetric {
             --region us-west-2
     fi
 }
+
+function forceFromSourceRun {
+    if [[ "$TEST_SUITE" == "src/__tests__/amplify-app.test.ts" ]]; then
+        echo "Not forcing run from source for $TEST_SUITE"
+        return
+    fi
+
+    if [[ "$TEST_SUITE" == "src/__tests__/datastore-modelgen.test.ts" ]]; then
+        echo "Not forcing run from source for $TEST_SUITE"
+        return
+    fi
```

Comment:
these two tests don't like from source run and fail with this
<img width="842" alt="image" src="https://user-images.githubusercontent.com/5849952/208966901-15de1ea0-42f2-42a2-af7b-f02f81a747e5.png">